5. Administrator Guides/File Upload/Minio.md

```diff
@@ -0,0 +1,47 @@
+# Minio
+
+* Minio is a distributed object storage server built for cloud applications and devops
+* You can use many as a replacement for the amazon s3 storage
+* a basic setup guide for a minio instance can you find here: http://docs.minio.io/
+* If you use docker, use the container image `minio/minio:edge` instead of latest (not included but needed fixes)
+
+* Go to your File Upload Settings in rocket.chat (administration -> File Uploads)
+* Adjust the following setting:
+
+`Storage Type: gridfs (default) -> amazon s3`
+
+* access the amazon s3 menu and adjust:
+
+```
+Amazon S3 Bucketname: <yourbucketname>
+Amazon S3 AWS AccessKeyID: <your minio access key>
+Amazon S3 AWSSecretAccessKey: <your minio secret access key>
+Region: us-east-1
+Bucket URL: https://minio.mydomain.com/<bucketname>
```

Comment:
The example for starting a minio server uses only http. Maybe this should be changed to:
+Bucket URL: http://minio.mydomain.com/<bucketname>